tutorials/hist/h1readanddraw.C

```diff
@@ -25,6 +25,7 @@
 
 void h1draw()
```

Comment:
The function name needs to follow the file renaming. I would suggest to improve readability for both file name and function name: `h1ReadAndDraw.C` or `h1_read_and_draw`. Pick whatever is more commonly used by other files in the same directory.

Reply:
Ah yes, I completely forgot about that.